package.json

```diff
@@ -34,7 +34,7 @@
   "devDependencies": {
     "asar": "^2.0.1",
     "electron-builder": "^22.3.2",
-    "electron-builder-squirrel-windows": "^21.2.0",
+    "electron-builder-squirrel-windows": "^22.3.3",
```

Comment:
Now you have a different point release here... Maybe a release is in progress, as I don't see it yet on https://github.com/electron-userland/electron-builder/releases.

Reply:
hmm, yes - better use the same one I guess